.github/workflows/build-test-smart-contracts.yaml

```diff
@@ -19,6 +19,17 @@ env:
   TARGET: wasm32-unknown-unknown
 
 jobs:
+  "wat-wasm-consistency":
```

Comment:
It did at some point, but it seems to have been removed.

The consistency script was moved from the node and I just altered it a bit.

As noted in the consistency script:
> The .wasm files are committed for convenience in testing, so that people
> who do not touch the scheduler or smart contracts do not need all the Wasm tooling installed.

The wasm files are used both in the node and in the integration testing lib. I don't want to add WABT in both to support compiling before testing. So I think this is the way to go for now.

Reply:
Yes, that is the way to go. As is pointed out in the script :) 

And the CI definitely did this at some point, but it seems this was removed with the move to github.

I can only find references in git to when we still had this on gitlab.

Reply:
Makes sense